ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/DataColumnSidecarRecoveringCustodyImpl.java

```diff
@@ -231,6 +259,7 @@ private void initiateRecovery(
                       .collect(Collectors.toUnmodifiableSet());
               final List<DataColumnSidecar> recoveredSidecars =
                   miscHelpers.reconstructAllDataColumnSidecars(block, sidecars, kzg);
+              totalDataAvailabilityReconstructedColumns.inc(recoveredSidecars.size());
```

Comment:
recovered is always 128, it's all. Are you sure you need this number for this counter? Maybe recovered.size() - existing.size()?